woocommerce/woocommerce-gutenberg-products-block
# Move `Chip` and `RemovableChip` to components package

## What

This PR is based on #11214 (`update/checkbox-component`)
Move `Chip` and `RemovableChip` to components package and update instances where they're imported:

- Active Filters
- Cart/Checkout coupons area

## Why

As more developers integrate their extensions with WooCommerce Blocks, the case for reusing components grows. This starts the ball rolling by creating a new package. Rather than recreating components developers can access wc.blocksComponents and get the ones we need.



## Testing Instructions

1. Create a coupon
2. Add the Active Filters block to a page, add the Filter by Attribute block, the Filter by Stock block, and the Products (Beta) block too.
3. Visit the page, apply some filters, ensure the chips show up correctly in the Active Filters block. Remove the filters by pressing the `x` on the chip.
<img width="344" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/5656702/37a83029-e111-4115-b68e-8810e5ac2d57">

4. Visit the Cart block. Apply the coupon you created in step 1. Ensure the coupon shows correctly:
<img width="248" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/5656702/30638e26-b0ed-48e0-b8ed-21ff3e880b54">

5. Remove the coupon.
6. Proceed to the Checkout block and re-apply the coupon, ensure it shows correctly and ensure it can be removed.



## Screenshots or screencast



N/A

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Make the `Chip` and `RemovableChip` components available in the `@woocommerce/blocks-components` package.
